pkg/adapters/tools_storage_adapter.go

```diff
@@ -22,6 +26,15 @@ type ToolsStorage interface {
 	Find(toolsFindOptions ToolFindOptions) (*manifestchecker.ToolSpec, error)
 }
 
+func NewToolNotFoundExError(toolName string) error {
+	return exerrors.New(fmt.Sprintf("tool with name [%s] not found", toolName))
+}
+
+func IsToolNotFoundExError(err error) bool {
+	errStack := exerrors.ErrorStack(err)
+	return strings.HasPrefix(errStack, "tool with name")
```

Comment:
Maybe better add a regex of the full message including the "not found" substring. Current check can be a bit ambiguous

Reply:
it is fully controlled by the function before. In fact I'd like to wrap go-errors to add a kind or something that can be checked without regexp, string contains etc

Reply:
Created issue #45 to solve this in next milestones

Reply:
okay 👍